.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: Continuous Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone @api3/promise-utils
+        uses: actions/checkout@master
+      - name: Check hyperlinks
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone @api3/promise-utils
+        uses: actions/checkout@v2
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
It might be better to have this run on a matrix of versions as this is a more general purpose library. Something like this:

https://github.com/andreogle/eslint-teamcity/blob/1ec5b2b9fab080dc4a599421257cde3ff01d00de/.github/workflows/continuous-build.yml#L21-L39

Reply:
I thought it's a bit overkill - but it doesn't hurt to add it.

Reply:
Btw. I think you have a typo in that repository. It should be `matrix.node-version` instead of `matrix.node`.

And it was actually a good call to add this, because the error messagefor TypeError changed between major node changes (which caused me some troubles).

Reply:
Thanks - yeah I noticed that typo as well and fixed it here: https://github.com/andreogle/eslint-teamcity/commit/4d5f2849fc8772c3a1a61a0127dd728f2409504a The CI for that repo has been only running against the latest Node version for _ages_ 😄